lib/routemaster/api_client.rb

```diff
@@ -72,15 +72,27 @@ def fget(url, **options)
     end
 
     def post(url, body: {}, headers: {})
-      _request(:post, url: url, body: body, headers: headers)
+      _wrapped_response _request(
+        :post,
+        url: url,
+        body: body,
+        headers: headers)
     end
 
     def patch(url, body: {}, headers: {})
-      _request(:patch, url: url, body: body, headers: headers)
+      _wrapped_response _request(
+        :patch,
+        url: url,
+        body: body,
+        headers: headers)
     end
 
     def delete(url, headers: {})
-      _request(:delete, url: url, body: nil, headers: headers)
+      _wrapped_response _request(
```

Comment:
If you really want to expose the same _class_ when deleting (honestly, I still don't see why) — then can you guarantee the response will be empty?

Maybe pass an empty body to the rapper, or soemthing?